.travis.yml

```diff
@@ -190,23 +209,24 @@ matrix:
     - name: nvcc-10.0 + clang-6 Debug
       env: ALPAKA_CI_DOCKER_BASE_IMAGE_NAME=ubuntu:16.04 CXX=clang++ CC=clang ALPAKA_CI_CLANG_VER=6.0.1 CMAKE_BUILD_TYPE=Debug   ALPAKA_CI_STDLIB=libstdc++ ALPAKA_CI_BOOST_BRANCH=boost-1.65.1 ALPAKA_CI_CMAKE_VER=3.11.4 ALPAKA_CI_SANITIZERS=      ALPAKA_ACC_GPU_CUDA_ENABLE=ON  ALPAKA_CUDA_VERSION=10.0 ALPAKA_CUDA_COMPILER=nvcc ALPAKA_CUDA_ARCH="70"
 
-    # HIP
+    ## HIP
     - name: HIP(nvcc8.0) + gcc-4.9 Debug
       env: ALPAKA_CI_DOCKER_BASE_IMAGE_NAME=ubuntu:16.04 CXX=g++     CC=gcc   ALPAKA_CI_GCC_VER=4.9     CMAKE_BUILD_TYPE=Debug   ALPAKA_CI_STDLIB=libstdc++ ALPAKA_CI_BOOST_BRANCH=boost-1.67.0 ALPAKA_CI_CMAKE_VER=3.11.4 ALPAKA_CI_SANITIZERS=      ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLE=OFF ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE=OFF ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE=OFF ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE=OFF ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE=OFF ALPAKA_ACC_CPU_BT_OMP4_ENABLE=OFF  ALPAKA_ACC_GPU_CUDA_ENABLE=OFF    ALPAKA_ACC_GPU_HIP_ENABLE=ON  ALPAKA_ACC_GPU_HIP_ONLY_MODE=ON  ALPAKA_CI_HIP_BRANCH="master" ALPAKA_HIP_PLATFORM=nvcc ALPAKA_HIP_ARCH="30;35" ALPAKA_CUDA_VERSION=8.0 ALPAKA_CUDA_COMPILER=nvcc
 
 branches:
     except:
-        - doc
+        - gh-pages
 
 cache:
     directories:
         - $ALPAKA_CI_DOCKER_CACHE_DIR
 
 script:
     - set -eovx pipefail
-    - sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install smem
-    - sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install moreutils
-    - ./script/travis/script.sh | ts
+    - if [ "$TRAVIS_OS_NAME" = "linux" ] ;then sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install smem ;fi
```

Comment:
you can make this in one line: `... install smem moreutils`

Reply:
I know. They have been added at different times and I still want to move the installation of `smem` to the place where it is really needed.